workflow-templates/pr_build.yaml

```diff
@@ -1,47 +1,39 @@
 ---
-name: Build & Tag Image on PR
-'on':
+name: Pull Request Build & Tag
+on:
   pull_request:
     branches:
-    - develop
-    - main
-    - master
+      - develop
+      - main
+      - master
     types:
-    - closed
-    - edited
-    - opened
-    - ready_for_review
-    - reopened
-    - synchronize
-    - workflow_dispatch
+      - opened
+      - reopened
+      - synchronize
+      - closed
 jobs:
-  docker_build:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Check Out GitHub Repo
-      if: github.event.pull_request.draft == false
-      with:
-        ref: "${{ github.event.pull_request.head.sha }}"
-      uses: actions/checkout@v2
-    - name: Check Source Branch
-      if: github.head_ref != 'develop' && (github.base_ref == 'master' || github.base_ref == 'main')
-      run: echo "PRs must be made to develop branch before merging to main/master"; exit 1
-    - name: Build And Push To Packages
-      if: github.event.pull_request.draft == false && github.event.action != 'closed' && github.event.pull_request.merged != true
-      env:
-        PR: "${{ github.event.pull_request.number }}"
-        SHA: "${{ github.event.pull_request.head.sha }}"
-        DOCKER_ACTOR: "${{ secrets.GHCR_USERNAME }}"
-        DOCKER_TOKEN: "${{ secrets.GHCR_TOKEN }}"
-      run: "curl https://raw.githubusercontent.com/kbase/.github/main/workflow-templates/scripts/pr_build.sh | bash"
-  tag_on_merge:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Tag Latest Image On Merge
-      if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
-      env:
-        PR: "${{ github.event.pull_request.number }}"
-        SHA: "${{ github.event.pull_request.head.sha }}"
-        DOCKER_ACTOR: "${{ secrets.GHCR_USERNAME }}"
-        DOCKER_TOKEN: "${{ secrets.GHCR_TOKEN }}"
-      run: "curl https://raw.githubusercontent.com/kbase/.github/main/workflow-templates/scripts/tag_on_pr_merge.sh | bash"
+  build-develop-open:
+    if: github.base_ref == 'develop' && github.event.pull_request.merged == false
+    uses: kbase/.github/.github/workflows/reusable_build.yml@main
+    secrets: inherit
+  build-develop-merge:
```

Comment:
Should the title of this action also contain the word "push"?

Reply:
Resolved this by changing the `name` field to `Pull Request Build, Tag, & Push`.
Keeping filename as `pr_build.yaml` for backwards compatibility.